pkg/skaffold/runner/v1/deploy.go

```diff
@@ -141,6 +143,12 @@ See https://skaffold.dev/docs/pipeline-stages/taggers/#how-tagging-works`)
 	event.DeployComplete()
 	eventV2.TaskSucceeded(constants.Deploy)
 	r.runCtx.UpdateNamespaces(namespaces)
+
+	// TODO(nkubala): implement status checking for docker deployments using ContainerWait
+	if r.localDeploy {
```

Comment:
I don't think we'll need `localDeploy` checks anymore since `deployer.GetStatusMonitor()` will return the correct status monitor.

Reply:
good point. will remove